src/modules/validation/validation-module-manager.js

```diff
@@ -7,24 +7,36 @@ class ValidationModuleManager extends BaseModuleManager {
 
     calculateRoot(assertion) {
         if (this.initialized) {
+            if (assertion === null) {
```

Comment:
In javascript parameters without a value can be null and undefined. To be sure you tested every case you can use this:
if (!!assertion) {

} else {
}
!assertion will return boolean true if assertion is null or undefined. And when you do !!assertion (generally ! is negation) you will get false. That's what you need.

Reply:
Hey Djole, thank you for the input. I implemented it as you said above, but Eslint reports the errors after committing.
I adopt the logic to be:

 getMerkleProof(assertion, index) {
        if (this.initialized) {
            if (!assertion) {
                throw new Error('Get merkle proof failed: Assertion cannot be null or undefined.');
            }
            return this.getImplementation().module.getMerkleProof(assertion, index);
        }
        throw new Error('Validation module is not initialized.');
    }